### DISCUSSION TITLE
Resume training of en_core_web_lg in spacy3

### DISCUSSION THREAD

**Discussion by nsorros** [explosion/spaCy#8598] [Help: Coding & Implementations] [answered]
Hey 👋

I am trying to replicate the behaviour of training an NER model starting from the weights of a base model like `en_core_web_lg` that in spacy2 we can we do with `spacy train en model_path train_data_path eval_data_path ---pipeline ner --base-model en_core_web_lg`.

I am initialising the config using the Quickstart guide https://spacy.io/usage/training and selecting en,ner,cpu and efficiency or accuracy (tried both). In the second it seems to initialise the vectors weights from `en_core_web_lg` but I think it still starts from a blank model.

Read this https://github.com/explosion/spaCy/discussions/7582 and some other answers I think the answer lies to `[initialize.before_init]`. Looking at the API https://spacy.io/api/architectures#parser and searching for the configuration label I have not been able to find what is the appropriate way to initialise the model so it starts from the pretrained weights. Any hints?

**Comment by adrianeboyd** [comment, marked as answer]
Hi, please have a look at this demo project, which shows how to update `ner` in an existing pipeline:

https://github.com/explosion/projects/tree/v3/pipelines/ner_demo_update

You can see the [`spacy project` docs](https://spacy.io/usage/projects) or the short intro here: https://github.com/explosion/spaCy/tree/master/examples/

(Be aware that this demo doesn't work with the `trf` models yet.)
